cloud-service-provider/aws/ec2-vm/terraform/README.md

```diff
@@ -0,0 +1,111 @@
+## 🤖 Terraform Example for AWS EC2 VM using the default VPC
+
+This example uses the following Terraform module:
+
+[Terraform Module](https://registry.terraform.io/modules/intel/aws-vm/intel/latest)
+
```

Comment:
Why don't we use the standard terraform ec2_instance module instead of Intel one? They seem to be almost the same.

Reply:
My point is that we should use vendor neutral [`ec2-instance` module](https://registry.terraform.io/modules/terraform-aws-modules/ec2-instance/aws/latest) from the Terraform community instead of Intel specific clone (`ec2-vm`) of the module. Or am I missing something here?

Reply:
The module being used is also an Open Source Community module. 
https://registry.terraform.io/modules/intel/aws-vm/intel/latest